atomic_physics/tests/test_utils.py

```diff
@@ -1,275 +1,205 @@
 import unittest
 
+import scipy.constants as ct
 import numpy as np
-
-from atomic_physics import polarization
+from atomic_physics.ions import ca40, ca43, ba137
+from atomic_physics.utils import (
+    ac_zeeman_shift_for_transition,
+    ac_zeeman_shift_for_state,
+)
 from atomic_physics.core import RFDrive
-from atomic_physics.ions import ba137, ca43
-from atomic_physics.utils import ac_zeeman_shift_for_state
+from atomic_physics.polarization import (
+    PI_POLARIZATION,
+    SIGMA_MINUS_POLARIZATION,
+    SIGMA_PLUS_POLARIZATION,
+)
+
+MU_B = ct.physical_constants["Bohr magneton"][0]

```

Comment:
IIRC we've used _uB for this elsewhere

Reply:
This is not really used in this file (left over from copying imports), will delete